src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transform-origin: center;
+    transition-duration: 0.3s;
```

Comment:
better indicate transition-name, or tipe it transition: trans-name trans-dur trans-func delay..
in your case it can be "transition: all 0.3s;" "transition: transform 0.3s, color 0.3s" - second is better to avoid mistakes